.env

```diff
@@ -0,0 +1,2 @@
+NODE_ENV=''
+BROWSER=''
```

Comment:
This is cool to have dotenv!

There's another var in there called `OFFLINE` ... not sure if we want it, but maybe just chuck in here so we're aware of it